platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -139,17 +139,33 @@ event_processor:
   context_path: /
   # The listening port of the event processing server
   port: 8084
+  # The flag to enable the status callback to the client domain
+  client_domain_status_callback_enabled: true
+  # The flag to enable the event delivery to the anchor business server
+  callback_api_event_delivery_enabled: true
```

Comment:
I don't love the naming here. I think something generalized would be better:

```
  events:
    transaction_status_updated:
      wallet_callbacks_enabled: true
      callback_api_requests_enabled: true
```

This way, we can add more event types and more handlers of those events using a predefined structure.

Reply:
There is a `event` section that defines the existing `EventService`. This is specific to `event-processor`. 

I would probably swap the structure like. Please refer to the updated commit.

Reply:
I see, I don't mind using the `event-processor` top-level object.

What do you think about using event type as a sub-object though? That way the business could subscribe to a subset of events, instead of having just a single `true`/`false` attribute for all event types.

For example,

```
event_processor:
  event_type:
    transaction_status_updated:
      client_callback_enabled: true
      callback_api_requests_enabled: true
    transaction_created:
      client_callback_enabled: false
      callback_api_requests_enabled: true
```

Reply:
I don't feel strongly about this. If we want to use a single attribute for all requests that it ok, we'll just have less flexibility and need to make assumptions about what anchors and clients will want.

Reply:
Not all callbacks are interested in all event types. How about we do this:
```
event_processor:
  # The context path of the event processing server
  context_path: /
  # The listening port of the event processing server
  port: 8084
  # The configuration of the status callback to the client domain
  client_domain_status_callback:
    enabled: true
  # The configuration of the event delivery to the anchor business server
  callback_api_event_delivery:
    enabled: true
```
This will leave more room for customization and we don't have to define the behavior of the events unless it is necessary to customize.

Reply:
Hey Jamie, can we use the following names, `client_status_callback` and `callback_api_requests`?

Reply:
Works for me!